Add read support for idle_time and low_battery_threshold

bno1 opened the pull request:
I tested the device files and the dbus interface with QDbusViewer and they work fine for my Basilisk X HyperSpeed.

Related issue: https://github.com/openrazer/openrazer/issues/1085

bno1 committed:
Add read support for idle_time and low_battery_threshold

Tested on Basilisk X HyperSpeed & Naga Epic Chroma.

z3ntu reviewed (approved):
LGTM, thank you! And @intact thanks for testing!

z3ntu committed:
Fix travis-ci build

Latest pylint supports isort 5.x.x but isort 5.x.x doesn't support
Python 3.5 (the version in Ubuntu 16.04) anymore so we have to
explicitely install isort 4.x.x.
